model/riscv_insts_zcb.sail

```diff
@@ -7,7 +7,7 @@
 /*=======================================================================================*/
 
 enum clause extension = Ext_Zcb
-function clause extensionEnabled(Ext_Zcb) = sys_enable_zcb()
+function clause extensionEnabled(Ext_Zcb) = sys_enable_zcb() & extensionEnabled(Ext_Zca)
```

Comment:
This is a separate behavioural change

Reply:
Good point. I can break that out into a separate PR.